nltk/parse/corenlp.py

```diff
@@ -328,7 +328,7 @@ def tokenize(self, text, properties=None):
 
         for sentence in result['sentences']:
             for token in sentence['tokens']:
-                yield token['originalText']
+                yield token['originalText'] or token['word']
```

Comment:
I would check here whether `originalText` or `word` are in token.

Actually, why is `originalText` unreliable? Maybe the extracted key should be a configuration option.